voxblox_ros/src/esdf_server.cc

```diff
@@ -135,6 +135,7 @@ void EsdfServer::updateMesh() {
 
 void EsdfServer::publishPointclouds() {
   publishAllUpdatedEsdfVoxels();
+  publishSlices();
```

Comment:
From memory we had a flag for if slices are published (if this wasn't a TsdfServer only thing), shouldn't that be checked here?